frontend/javascripts/admin/voxelytics/ai_model_list_view.tsx

```diff
@@ -0,0 +1,151 @@
+            defaultSortOrder: "descend",
+            sorter: (a: AiModel, b: AiModel) => a.created - b.created,
+            render: (model: AiModel) => <FormattedDate timestamp={model.created} />,
+          },
+          {
+            title: "User",
+            dataIndex: "user",
+            key: "user",
+            render: (user: AiModel["user"]) => formatUserName(activeUser, user),
+            filters: _.uniq(aiModels.map((model) => formatUserName(null, model.user))).map(
+              (username) => ({
+                text: username,
+                value: username,
+              }),
+            ),
+            onFilter: (value: string | number | boolean, model: AiModel) =>
+              formatUserName(null, model.user).startsWith(String(value)),
+            filterSearch: true,
+          },
+          {
+            title: "Status",
+            dataIndex: "trainingJob",
+            key: "status",
+            render: (trainingJob: AiModel["trainingJob"]) =>
+              trainingJob && <JobState job={trainingJob} />,
+          },
+          {
+            title: "Comment",
+            dataIndex: "comment",
+            key: "comment",
+          },
+          {
+            title: "",
+            render: (model: AiModel) => {
+              if (model.trainingJob == null) {
+                return;
+              }
+              const { voxelyticsWorkflowHash, trainingAnnotations } = model.trainingJob;
+
+              return (
+                <div>
+                  {voxelyticsWorkflowHash != null ? (
+                    <Link to={`/workflows/${voxelyticsWorkflowHash}`}>Voxelytics Report</Link>
+                  ) : null}
+                  {trainingAnnotations == null ? null : trainingAnnotations.length > 1 ? (
+                    <a
+                      href="#"
+                      onClick={() => {
+                        Modal.info({
+                          content: (
+                            <div>
+                              The following annotations were used during training:
+                              <ul>
+                                {trainingAnnotations.map(
+                                  (annotation: { annotationId: string }, index: number) => (
+                                    <li>
+                                      <a
+                                        href={`/annotations/${annotation.annotationId}`}
+                                        target="_blank"
+                                        rel="noreferrer noopener"
+                                      >
+                                        Annotation {index + 1}
+                                      </a>
+                                    </li>
+                                  ),
+                                )}
+                              </ul>
+                            </div>
+                          ),
+                        });
+                      }}
```

Comment:
Nitpicky: Can this be moved into a separat method? It blows up the return statement. 

But feel free to ignore, as it is not so relevant :)